File: C++ Works/faktoriyel.cpp
#include <iostream>
int main() {
    int sayi;
    int sonuc=1;
    std::cout<<"faktoriyel hesaplama programi\n";
    std::cout<<"hesaplamak istediginiz sayiyi giriniz:  ";
    std::cin>>sayi;
    if (sayi<0) {
        std::cout<<"0 dan kucuk bir deger girmeyiniz.";

    }else {
        for (int i = 1; i <= sayi; i++) {
            sonuc*=i;
        }
        std::cout<<"sonuc: "<< sonuc;
    }
    return 0;
}

File: python works/faktoriyel.py
sonuc=1
try:
    sayi = int(input("bir sayi giriniz: "))
    if sayi<0:
        print("0 dan kucuk deger girmeyin..")
    else:
        for i in range(1,sayi):
            sonuc *=i
    print("faktoriyel sonucu: ",sonuc)
except ValueError:
    print("bir sayi degeri girein string deger girmeyin..")
finally:
    print("sistemden cikildi")
